src/Utils.ts

```diff
@@ -28,6 +23,22 @@ const DEFAULT_MANAGER_EVENTS: DefaultManagerEvent[] = [
   'TransactionRelayedButRevertedByRecipient',
 ];
 
+const CONFIG_CONTRACTS = 'contracts';
+const CONFIG_BLOCKCHAIN = 'blockchain';
+const CONFIG_APP = 'app';
+const CONFIG_RELAY_HUB_ADDRESS = 'relayHubAddress';
+const CONFIG_RSK_URL = 'rskNodeUrl';
+
+const getRelayHub = (): RelayHub => {
+  const provider = getProvider();
+
+  const relyHubAddress = config.get<string>(
+    `${CONFIG_CONTRACTS}.${CONFIG_RELAY_HUB_ADDRESS}`
+  );
+
+  return RelayHub__factory.connect(relyHubAddress, provider);
+};
+
```

Comment:
What about something link this:
```suggestion
const getConfiguredRelayHubAddress = () => config.get<string>(
  `${CONFIG_CONTRACTS}.${CONFIG_RELAY_HUB_ADDRESS}`
);

const getRelayHub = (provider = getProvider(), relayHubAddress = getConfiguredRelayHubAddress()): RelayHub => RelayHub__factory.connect(relayHubAddress, provider);
```

Reply:
Implemented those changes, the functionality is still the same

Reply:
updated